source/account_structure/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Account structure 
+weight: 55
+---
+
+# Account structure 
+
+If you have multiple services integrated, you can treat them separately or as
+a combined single service. You can use the GOV.UK Pay platform to set this this within
+GOV.UK Pay and within your Payment Service Provider (PSP). 
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/account_structure.png)
+
+You may have 2 services, for example for parking permits and parking fines.
+The diagram takes this example and illustrates the different possible
+structures you can have:
+
+### Type 1 (GOV.UK Pay separate, PSP separate)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+both GOV.UK Pay and the PSP.
+
+### Type 2 (GOV.UK Pay combined, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as a single combined
+service in GOV.UK Pay and in the PSP.
+
+### Type 3 (GOV.UK Pay separate, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+GOV.UK Pay and as a single combined service in the PSP. 
+
+## Editing GOV.UK Pay services 
+
+Each service defined in GOV.UK Pay has its own API.
```

Comment:
is it right to say 'own API' - or own API key?

Reply:
I was confused about this—this is from the copy in the original slides from Miriam I think? I don't think this is an accurate statement.

Reply:
The only thing I can think this is intended to be is “API keys” or something to convey that each service is separated at the API level i.e. you only ever work with one service at a time via the API. (In actuality, the separation is at the gateway account level: in our system, there are services and each service has one or more gateway accounts — usually a live account and a test sandbox account — and one works with a gateway account at the API level. However, in user-facing docs, we tend to only use the term “gateway account” when talking specifically about a gateway account with a PSP, eliding our sense of gateway accounts into services.)

Reply:
I think in that case (thank you @alexbishop1), bearing it mind that the gateway account interaction is implied, is it sufficient to simply say(?):

> Each service defined in GOV.UK Pay has its own API key.

Or do we need to be more specific about how each service is separated at the gateway account level, in addition?